tests/integration/test_branch_integration.py

```
@@ -0,0 +1,369 @@
+        
+        assert len(workflow_files) > 0, "No workflow files found"
+        
+        for wf_file in workflow_files:
+            try:
+                with open(wf_file, 'r') as f:
+                    workflow = yaml.safe_load(f)
+                
+                assert workflow is not None
+                assert isinstance(workflow, dict)
+                assert 'jobs' in workflow
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)